OODebug/src/debugger/JavaDebugger.cpp

```diff
@@ -69,6 +69,22 @@ struct VariableObserver {
 		QList<Probes::ValueCalculator> valueCalculators_;
 };
 
+struct UserVisibleBreakpoint {
+		qint64 breakpointId_;
```

Comment:
Is there any specific reason why breakpoints are a 64 bit int? why not just `int`?